### ISSUE TITLE
As a user I would like to know which article is big/small

### ISSUE THREAD

**Issue by ggajos** [jvm-bloggers/jvm-bloggers#361]
Since there are more and more links every week, it would be really great to separate "short notes" from really long reads. I think it can be done by showing a number of words of the article. Or, even better, time to read (like on medium pages https://medium.com/). Time to read can be calculated as *words/words per minute constant".

However, any other ideas welcomed.

**Comment by tdziurko** [comment]
Time to read looks really promising so +1 from my side for this feature :)

**Comment by airborn** [comment]
The only problem I can see is that some sources only includes short part of the posts in RSS and we will need to fetch the whole webpage and parse it as well.

**Comment by tdziurko** [comment]
@airborn Yes, that is how I think it should be implemented to be really useful. But unfortunately this brings much more complexity: parsing blog post, checking if we don't include other blog posts that might lurk below current one, etc. etc.

Do you @ggajos had any initial idea how you would like it to be implemented? Any thoughts, approaches?

**Comment by ggajos** [comment]
@tdziurko I think that scraping is too much work (and not reliable at all). I suggest sticking to just counting words from RSS.

@airborn do you have maybe a link to the RSS where no full content is available? I reviewed some of them but seems like all are ok. If this is only a small margin, then maybe we can suggest to those bloggers to update the RSS format? Ideally, if we would be able to tell if the content is cut or not, so we're not displaying the number there at all.

Another approach.

We can assume that we're displaying the number only if the read time is more than 5 minutes. It means if there is no number displayed, then it's either: unknown (because the content is cut) or less than 5 mins. Seeing that article is longer that 5 mins are already a nice value I think.

Thanks for the attention :).

**Comment by airborn** [comment]
Peoples are displaying only part of the content deliberately - to enforce you to visit their website, and display ads. Guess that asking them to stop this is pointless. 

As an example please take a look at: http://blog.codeleak.pl/feeds/posts/default?alt=rss

**Comment by ggajos** [comment]
Thanks @airborn. Then, I think the "Another approach" is still valid. We're not going to show any numbers for those blogs.

(More advanced scenario)
I'm just thinking here about the minimal version of this feature. If we would like to invest more time, then we might want to scrape the content of the blog site but use some simple rule instead of assumptions. For example, scrape the page and look for `<article>` tag only. Then, we might document that having this *tag* on the blog page is going to add time to read.

**Comment by tdziurko** [comment]
As a MVP we could start with checking RSS data and calculate time basing on this. Then, if we see that it's worth to move into more detailed algorithm, we could experiment with some more sophisticated algorithm checking actual website with a blog post.
